app/konnect/updates.md

```diff
@@ -11,7 +11,10 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## March 2024
 
-**Enhanced error notification in {{site.konnect_short_name}}**
+**Refined user experience for Control Plane Groups**
+: We've made improvements to refine the end-to-end workflow of Control Plane Groups (CPGs) for our customers. Now, CPGs are more intuitive, and easier to understand.
```

Comment:
I would suggest not shortening in the release note. Just spell out Control Plane Groups

Reply:
@rosskukulinski done